loaders/src/glTF/2.0/Extensions/KHR_materials_sheen.ts

```diff
@@ -86,6 +87,12 @@ export class KHR_materials_sheen implements IGLTFLoaderExtension {
             }));
         }
 
+        if (properties.roughnessFactor !== undefined) {
+            babylonMaterial.sheen.roughness = properties.roughnessFactor;
```

Comment:
Should probably add the default value in the else case to prevent reusing the model roughness in gltf by default ?

Reply:
Change performed